cmd/gocat/main.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Go Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Originally authored by Russ Cox (https://code.google.com/p/rsc/source/browse/cmd/bundle/main.go)
+//
+// Modified by Joseph Naegele, 2015
+
+// Gocat combines multiple Go source files into a single source file,
+// optionally adding a prefix to all top-level names.
+//
+// Usage:
+//	gocat [-p pkgname] [-x prefix] file_0 [file_n...] >file.go
+//
+// Example
+//
+//	gocat -p acl github.com/naegelejd/go-acl/acl* > acl.go
+//
+// Bugs
+//
+// Gocat has many limitations, most of them not fundamental.
+//
+// It does not work with cgo.
+//
+// It does not work with renamed imports.
+//
+// It does not correctly translate struct literals when prefixing is enabled
+// and a field key in the literal key is the same as a top-level name.
+//
+// It does not work with embedded struct fields.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/itsmontoya/gocat"
```

Comment:
I know this is for your fork, but could you change it to `naegelejd` for the merge? Thanks!